README.md

```diff
@@ -1,38 +1,28 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# NBC Webapp, built for the latest features.
+### Please note that this is still a work-in-progress and that changes will be made until a stable deployment phase has been reached.
 
-## Getting Started
+## Architecture and threat model
+![image](https://user-images.githubusercontent.com/60882255/235132605-0acd2780-5eeb-4acb-99d8-0d0693094afd.png)
 
-First, run the development server:
+## Website
+Please have a Metamask wallet ready to access the web app.
+[Click here to visit the live web app](https://nbc-webapp.vercel.app/)
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-```
+## Local development
+If you prefer to instantiate a local host instance of the web app instead, please do the following:
+### 1. Clone and visit the project
+`git clone https://github.com/NotBoringCompany/nbc-webapp.git`
```

Comment:
not sure why these are marked as changes? Nothing was changed... and its compared against the old `README` 🤷‍♂️ .

Github's bugging.